### PR TITLE
fix the creation of workpackages when a validation-error occurs

### PR DESCRIPTION
**Pull request by mugwump**
added missing :locals to the render-call of create; fixed the highlighting in the menue for workpackages. (see [#1789](https://www.openproject.org/issues/1789))

### CODE REVIEW

**Comment by mugwump** [comment]
this one is cleaner, contains only the three files that are actually involved.

**Comment by ulferts** [comment]
Addes some commits to fix the highlighting of work package summary and to remove the "View all work packages" menu item.

**Comment by ulferts** [comment]
This should also fix the jump to as mentioned in https://www.openproject.org/issues/1418

File: features/work_packages/reset_filters.feature [hunk 1]
```diff
@@ -36,14 +36,6 @@ Feature: Resetting filteres on work packages
...
   @javascript
-  Scenario: Clearing filters via the menu
-    When I toggle the "Work packages" submenu
-     And I follow "View all work packages"
-
-    Then I should be on the work package index page of the project called "project1"
-     And I should see "Some issue"
-
-  @javascript
   Scenario: Clearing filters via the filter buttons
```
Comment (mugwump) [inline features/work_packages/reset_filters.feature:39]: Is this intentional or did it slip in? Can't see the relation to the other changes...

**Pull request merged**